mathesar_ui/src/stores/table-data/meta.ts

```diff
@@ -218,9 +218,15 @@ export class Meta {
             value: term.value,
           }));
           filter[$filter.combination.id as string] = terms;
-          params.push(
-            `filters=${encodeURIComponent(JSON.stringify(filter))}`,
-          );
+          if ('and' in filter && filter.and.length > 0 && filter.and[0].op === 'get_duplicates') {
+            params.push(
+              `filters=${encodeURIComponent(JSON.stringify([filter.and[0]]))}`,
+            );
+          } else {
+            params.push(
+              `filters=${encodeURIComponent(JSON.stringify(filter))}`,
+            );
+          }
```

Comment:
This is the only region where I'd like the special handling to be present. Once the bug in the backend is fixed, it would be easier for us to revert this change.